posts/2018-11-28-http4s-error-handling-mtl-2.md

```diff
@@ -0,0 +1,456 @@
+---
+layout: post
+title: Error handling in Http4s with classy optics – Part 2
```

Comment:
Please use a title that's consistent with Part 1 ("Http4s error handling with Cats Meow MTL"). Also, it would be great to have a link from Part 1 to here.

Reply:
Do you think it's okay to update the title of the first part? I like this one more :smile: 

Do you mean adding a link to the title? How can I do that?

Reply:
> Do you think it's okay to update the title of the first part? I like this one more

No problem, you can add the change to this PR.

> Do you mean adding a link to the title? How can I do that?

I was thinking in the body text of the post. In this post, you have a back link somewhere in the beginning. You can do the same for the old post, but in the other direction.

Reply:
Got it, will do soon! :+1: